### ISSUE TITLE
[NSFW] pornlib.com

### ISSUE THREAD

**Issue by FadeMind** [AdguardTeam/AdguardFilters#43252]
**Site Sample** | 
----------------|
https://www.pornlib.com/

Information                            | Value
---                                    | ---
Operating system:                      | Windows 10 1909 x64 (lastest)
Browser:                               | Google Chrome Stable x64 (lastest)
AdGuard version:                       | **7.3 nightly X (XXXX)** 
Stealth Mode:					       | disabled
Chrome Extension: | [Polska Ciasteczkowa Zgoda - extra JS scripts blocking EU GDPR annoyances on polish websites](https://chrome.google.com/webstore/detail/polish-cookie-consent/bniijddcmabghibaojbkbnngbedopbno) 
Chrome Extension: | [AdGuard Adblocker Extension - Integration Mode](https://chrome.google.com/webstore/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg) 

| **=== FILTERS ENABLED ===**
|---|
| **SECTION :: Other**
| CoinBlockerLists by ZeroDot1
| FuckFuckAdblock
| **SECTION :: Ads**
| Adguard Base Filter 
| Peter Lowe's List
| **SECTION :: Privacy**
| Adguard Spy Filter
| **SECTION :: Social**
| Adguard Social Filter
| **SECTION :: Annoyances**
| Adguard Annoyances
| I don't care about cookies
| Fanboy's Cookiemonster
| **SECTION :: Security**
| Malware Domains
| Spam404
| **SECTION :: Customs**
| AlleBlock
| KAD - Przekręty
| Official Polish filters for AdBlock, uBlock Origin & AdGuard
| Polish Anti Adblock Filters
| Polskie Filtry Elementów Irytujących
| Polskie Filtry Społecznościowe

### Issue:

Ads, popups

<details><summary><b>Screenshot:</b></summary>

![scr-1572503623-001](https://user-images.githubusercontent.com/9846948/67924396-d5207200-fbb0-11e9-9567-f1c3bb4b1ffd.png)

</details><br/>

@AdamWr @Alex-302

**Comment by FadeMind** [comment]
@AdamWr FFA robiło za dużo zamieszania na niektórych stronach - więc go usunąłem z konfiguracji i problemu z adbox na tej stronie nie ma. Poczekam na odświeżenie głównych filtrów i potwierdzę ci czy jest już wszystko ok - później.

**Comment by FadeMind** [comment]
@AdamWr  Jedynie Adguard Popup blocker dziwnie się zachowuje po kliknięciu na dowolny link do filmu. Mianowicie na sekundę pokazuje się informacja, że zablokowano popup, a następnie przekierowuje prawidłowo do podstrony z linku. Nie jestem w stanie wychwycić jaki popup jest blokowany.

**Comment by AdamWr** [comment]
Tak, zgadza się, ale jak wyłączysz `AdGuard Popup blocker`, to wyskakujące okienko powinno być zablokowane przez jedną z dodanych reguł, więc myślę, że w tym przypadku to nie jest jakiś duży problem.
